packages/sdk/tests/e2e/dataProtectorCore/grantAccess.test.ts

```diff
@@ -98,6 +98,25 @@ describe('dataProtectorCore.grantAccess()', () => {
     MAX_EXPECTED_WEB2_SERVICES_TIME
   );
 
+  it(
+    'Should be able to authorize specific user when `any` is already authorized',
+    async () => {
+      // grantAccess to any user
+      await dataProtectorCore.grantAccess({
+        ...input,
+        authorizedUser: ethers.ZeroAddress,
+        authorizedApp: sconeAppAddress,
+      });
+      // grantAccess to specific user
+      await dataProtectorCore.grantAccess({
+        ...input,
+        authorizedApp: sconeAppAddress,
+      });
+      expect(true).toBe(true);
+    },
+    2 * MAX_EXPECTED_WEB2_SERVICES_TIME
+  );
+
   it(
```

Comment:
add tests for scenario 2 (pass isAppStrict as a parameter to the function `getGrantedAccess`)

Reply:
add test for displaying message : 

`An access has been already granted to the user: ${vAuthorizedUser} with the app: ${vAuthorizedApp}`

Reply:
`getGrantedAccess` should not have `isAppStrict` as new param => discuss this morning

Reply:
Add new test to test error message thrown : 
Commit: 1306f3cb751c7f23e2103b1ef3d4625b5cd3db97